lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -609,13 +610,18 @@ def _section_send_email(course, access):
         # xblock rendering.
         request_token=uuid.uuid1().get_hex()
     )
+    cohorts = []
+    if is_course_cohorted(course_key):
+        cohorts = [cohort for cohort in get_course_cohorts(course)]
     email_editor = fragment.content
     section_data = {
         'section_key': 'send_email',
         'section_display_name': _('Email'),
         'access': access,
         'send_email': reverse('send_email', kwargs={'course_id': unicode(course_key)}),
         'editor': email_editor,
+        'cohorts': cohorts,
+        'default_cohort_name': DEFAULT_COHORT_NAME,
```

Comment:
This is what @cahrens and I decided to do for handling the special casing of default cohorts. If a cohort with this name exists, the course authors may not recognize the name, so extra text describing it is needed.

Users without any cohort are not included.